Update for Vite 4

Is this compatible with Vite 4?

Can we just bump the peerDependency or are there other changes that need to be made first?

The CLI excerpt from `npm i --save-dev sass` is this (happens presumably with any package):

![CLI excerpt](https://user-images.githubusercontent.com/70766223/206868593-edee7f07-74aa-4432-b9f6-e245b29046be.png)

In text form:
```ps1
PS C:\Users\Keyacom\coding\fire-visualize> npm i --save-dev sass
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @preact/preset-vite@2.4.0
npm ERR! Found: vite@4.0.0
npm ERR! node_modules/vite
npm ERR!   peer vite@">=2.0.0-beta.3" from @prefresh/vite@2.2.9
npm ERR!   node_modules/@prefresh/vite
npm ERR!     @prefresh/vite@"^2.2.8" from @preact/preset-vite@2.4.0
npm ERR!     node_modules/@preact/preset-vite
npm ERR!       dev @preact/preset-vite@"^2.4.0" from the root project
npm ERR!   dev vite@"^4.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"2.x || 3.x" from @preact/preset-vite@2.4.0
npm ERR! node_modules/@preact/preset-vite
npm ERR!   dev @preact/preset-vite@"^2.4.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vite@3.2.5
npm ERR! node_modules/vite
npm ERR!   peer vite@"2.x || 3.x" from @preact/preset-vite@2.4.0
npm ERR!   node_modules/@preact/preset-vite
npm ERR!     dev @preact/preset-vite@"^2.4.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\Keyacom\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Keyacom\AppData\Local\npm-cache\_logs\2022-12-10T17_20_38_267Z-debug-0.log
```

To fix this, I set `devDependencies.vite` in `package.json` to `^3.2.1 <4.0.0`, and it works properly.

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
 	},
 	"peerDependencies": {
 		"@babel/core": "7.x",
-		"vite": "2.x || 3.x"
+		"vite": "2.x || 3.x || 4.x"
 	},
 	"devDependencies": {
 		"@babel/core": "^7.15.8",

